Aptoide/aptoide-client-v8
# [APP-346] AAB install time

**What does this PR do?**

This PR aims at implementing the request and instalation of dynamic splits install time.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] DynamicSplitsManager.java

**How should this be manually tested?**

You can try and download any app to check that the request is being made.
In order to confirm that it is not always returning empty (which should be most of the cases for the moment), you can ask me for a list of apps that already have these splits added to them. After that, confirm that all splits were installed.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-346](https://aptoide.atlassian.net/browse/APP-346)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
